src/domain/participant/index.js

```diff
@@ -59,7 +58,6 @@ const ParticipantNotFoundText = 'Participant does not exist'
 const create = async (payload) => {
   try {
     const participant = await ParticipantModel.create({ name: payload.name })
-    if (!participant) throw new Error(CreateParticipantGenericErrorText)
```

Comment:
As per our discussion. This scenario will never occur since the create as an insert and any errors will be handled by the `ParticipantModel`.

Reply:
true